cmd/migrations.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package cmd
```

Comment:
I'm not sure this is the right package since this isn't really a command. Maybe make a migrations package in `internal`?

Reply:
Agreed.Placing the README there.